packages/orm/src/Model.ts

```diff
@@ -432,6 +432,7 @@ export default class Model<MClassType extends typeof AnyModel = typeof AnyModel,
    *
    * @return {Object} a reference to the plain JS object in the store
    */
+  //@ts-ignore
```

Comment:
Temporarily ignoring an error here - as we talked during the last sync, I will probably need to make a breaking change here - start using `getRef` fn instead of a getter (the typing issue inside model classes).